packages/google_sign_in/google_sign_in_ios/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -84,32 +84,45 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
                                  message:@"Games sign in is not supported on iOS"
                                  details:nil]);
     } else {
+      NSMutableDictionary<NSString *, NSString *> *plist = nil;
```

Comment:
nits: 
It is unnecessary to init as nil, simply:
`NSDictionary<NSString *, NSString *> *plist`

It seems to me plist doesn't need to be mutable.

Reply:
Done